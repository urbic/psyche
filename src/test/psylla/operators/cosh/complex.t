0. 1. complex cosh ?
