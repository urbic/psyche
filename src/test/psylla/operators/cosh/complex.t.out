0.5403023058681398+0.0*i
